### ISSUE TITLE
Popup enforces "btn-sm" for custom buttons

### ISSUE THREAD

**Issue by kikonen** [mistic100/Bootstrap-Confirmation#104]
I'm trying to use custom buttons, but effect is not desired since following styling enforcedly applied always

```
var BTN_CLASS_DEFAULT = 'btn btn-sm h-100 d-flex align-items-center';
....
    _proto._setCustomButtons = function _setCustomButtons($tip) {
      var self = this;
      var $group = $tip.find(Selector.BUTTONS).empty();
      this.config.buttons.forEach(function (button) {
        var btn = $('<a href="#"></a>').addClass(BTN_CLASS_DEFAULT).addClass(button.class || 'btn btn-secondary').html(button.label || '').attr(button.attr || {});
...
```

i.e. buttons are enforced to be "btn-sm"

**Comment by kikonen** [comment]
okey, seemingly with custom buttons that is right now not doable, for "standard buttons" via changing "template" seemingly possible to make it work.

However, requires then manually encoding extra data attributes & al. directly into template.

**Comment by mistic100** [comment]
@kikonen Can you give a try to this commit (not published on NPM).

I reworked how the classes are applied on buttons, this will be published as 4.1.0.
Now every custom buttons classes (for standard and custom buttons) must contain the "btn" class and the "btn-sm" if one want to have smaller buttons.
